draft-ietf-jose-hpke-encrypt.md

```diff
@@ -207,12 +264,74 @@ In HPKE JWE Key Encryption:
 - The protected header MUST contain an "enc" that is registered in both the IANA HPKE AEAD Identifiers Registry, and the IANA JSON Web Signature and Encryption Algorithms Registry.
 - The recipient unprotected header parameters "psk_id" and "auth_kid" MAY be present.
 - The recipient unprotected header parameter "ek" MUST be present.
-- The recipient unprotected header parameter MAY contain "alg" and "enc" values.
+- The recipient unprotected header MUST contain a registered HPKE "alg" value.
+- The recipient unprotected header MUST contain an "enc" and it must be set to the value "dir".
```

Comment:
In the case of Key Encryption, what is the need to convey "enc" in the recipient unprotected header? I don’t see a need for it in this mode, as the HPKE JWE Key Encryption Mode can function similarly to Key Agreement with ECDH-ES.